tests/capture/input_generators/test_attack_point_iterator.py

```diff
@@ -1,3 +1,16 @@
+# Copyright 2024 Google LLC
```

Comment:
QQ I have seen a lot of "Copyright 20CD-2024" Is that the preferred style, both are ok, or this one is preferred?

Reply:
When you first put the header, you put the current year. When you edit the file, you change the header to `20xx-20yy` with `yy` being the current year.
Does that answer your question?

Reply:
Totally, both is fine with me.